stripe/src/main/java/com/stripe/android/view/CardNumberEditText.kt

```diff
@@ -139,6 +140,7 @@ class CardNumberEditText internal constructor(
     internal var isProcessingCallback: (Boolean) -> Unit = {}
 
     init {
+        inputType = InputType.TYPE_CLASS_NUMBER
```

Comment:
Not too familiar with these APIs, but just ensuring -- does this block decimal inputs?

Reply:
Good call out - I verified that it behaves as intended